### ISSUE TITLE
508-defect-3 [COGNITION]: COVID Vaccine Trials - None of the above and Prefer not to answer checkbox "uncheck all" behavior should be clear to users

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#18271]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a user with a cognitive impairment, I might not understand the "None of the above" or "Prefer not to answer" checkboxes will unselect all previous checked boxes. This could be a frustrating experience or cause users to abandon the form.

## Details

The "None of the above" and "Prefer not to answer" checkboxes should have a visual and semantic separation from the other checkboxes. They should also make it clear to all users that checking these special boxes will unselect all other boxes in that fieldset. This will help prevent errors in submission and improve the user experience.

## Acceptance Criteria

- [ ] "None of the above" and/or "Prefer not to answer" checkboxes are visually separated from other checkboxes in the fieldset
- [ ] Uncheck all checkboxes are clearly labeled
- [ ] Uncheck all boxes announce their function to screen readers

## Environment

* https://www.va.gov/coronavirus-research/volunteer/sign-up

## WCAG or Vendor Guidance (optional)

* [On Input: Understanding SC 3.2.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html)

## Screenshots or Trace Logs

![uncheck-all](https://user-images.githubusercontent.com/934879/103952556-3bcee300-5106-11eb-999d-8a90f3c019c5.gif)

**Comment by artsymartha68** [comment]
We discussed this at the Design System Council meeting on Friday. The consensus was to rework the questions so that the checkboxes could behave normally. So asking an enabling question to start using radio buttons, and if Yes is selected, then displaying/enabling the checkboxes so the user can multi-select. 

For this it might be
( ) None
( ) Prefer not to answer
( ) Yes
(and then Yes enables all the checkbox options below it)

<img width="458" alt="Screen Shot 2021-01-22 at 10 50 38 AM" src="https://user-images.githubusercontent.com/47792423/105858425-4ec42d00-5fb9-11eb-877c-46a5169469b7.png">
